acquia_lift.routing.yml

```diff
@@ -4,4 +4,47 @@ acquia_lift.admin_settings_form:
     _form: '\Drupal\acquia_lift\Form\AdminSettingsForm'
     _title: 'Acquia Lift settings'
   requirements:
-    _permission: 'administer acquia lift'
+    _permission: 'administer acquia_lift'
+
+acquia_lift.slot.overview:
+  path: '/admin/config/content/acquia-lift/slots'
```

Comment:
It seems Drupal 8 core has the naming all over the place, for example [block](https://github.com/drupal/drupal/blob/6b95f9a8503b5b3a45e8e45b0805ee31f879fd9b/core/modules/block/block.routing.yml) and [block_content](https://github.com/drupal/drupal/blob/71142958f963e7cd6c3132e6337e8406fa2a17e3/core/modules/block_content/block_content.routing.yml).

So I guess it is just a matter of personal preference here. My only recommendation would be that we should use "slot" all across (and no "slots"), as the core conversions suggest.

Otherwise, my personally (and definitely biased) preference would go for "acquia_lift.slot.overview" and "/admin/config/content/acquia-lift/slot/overview", and avoid defining or using "/admin/config/content/acquia-lift/slot" completely (which I hope is possible). But again, personal preference is for your consideration but not an ask.